src/index.js

```diff
@@ -79,7 +80,7 @@ function ParseServer({
   cloud,
   collectionPrefix = '',
   clientKey = '',
-  javascriptKey = '',
+  javascriptKey = randomString(20),
```

Comment:
The issue was that when you omit the javascriptKey the parse SDK gets initialized wig an '' and doesn't like it when communicating with cloud code. If a user don't pass a JS key, the it means he doesn't want to use the JS SDK, so I init it to a random string. Actually, because it's init as an empty string, all the JS calls would have failed already as it would expect a javascriptKey header to be set to something empty and not undefined.